charts/signoz/templates/frontend/config.yaml

```
@@ -9,7 +9,9 @@ data:
     server {
       listen       {{ .Values.frontend.service.port }};
       server_name  _;
-
+{{- if .Values.frontend.customConfig }}      
+      {{ .Values.frontend.data | nindent 6 }}
+{{- end }}
```

Can you use `nginxExtraConfig` instead?

```suggestion
    {{- with .Values.frontend.nginxExtraConfig }}
      {{ . | nindent 6 }}
    {{- end }}
```

---

Of course, it makes more sense 👍